spec/Menu_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'Menu'
+
+describe Menu do
+
+	subject(:menu) {described_class.new}
+	
+	it 'Should allow you to see the menu' do
+    expect(subject.see_menu).to eq Menu::DEFAULT_MENU
+  end
```

Comment:
Lint/BlockAlignment: `end` at 9, 2 is not aligned with `it 'Should allow you to see the menu' do` at 7, 1.